pycode/test/test_epidata_get_population_data.py

```diff
@@ -320,7 +316,36 @@ def test_load_age_population_data(self, mock_read_excel1, mock_read_excel2, mock
         pd.testing.assert_frame_equal(reg_key_read, reg_key_write, check_dtype=False)
         pd.testing.assert_frame_equal(zensus_read, zensus_write, check_dtype=False)
 
-# TODO: How to test hdf5 export?
+    @patch('epidemiology.epidata.getPopulationData.gd.loadCsv')
+    @patch('epidemiology.epidata.getPopulationData.pandas.read_json')
+    @patch('epidemiology.epidata.getPopulationData.gd.loadExcel')
+    def test_errors(self, mocklexcel, mockrjson, mocklcsv):
+        mockrjson.side_effect = ValueError
+        [read_data, file_format, out_folder, no_raw] = [True, "json", self.path, True]
+
+        directory = os.path.join(out_folder, 'Germany/')
+        gd.check_dir(directory)
+        with self.assertRaises(SystemExit) as cm:

```

Comment:
functions should never call exit, I think. Errors should be reported properly (in python usually by raising an exception of the right type) so that users of the function can handle it. 

Maybe this should be a separate issue since it's happening all over.